src/particles.cpp

```diff
@@ -37,6 +133,20 @@ void ParticleParameters::serialize(std::ostream &os, u16 protocol_ver) const
 	writeU16(os, node.param0);
 	writeU8(os, node.param2);
 	writeU8(os, node_tile);
+	writeV3F32(os, drag);
+	jitter.serialize(os);
+	bounce.serialize(os);
```

Comment:
textures are serialized in server.cpp at 1667 (legacy texture) and 1686 (texpools)

Reply:
in regards to blend parameters not working, it sounds like that's probably a problem with your GPU or how irrlicht is talking to your GPU, because i've had no problems testing blend effects on mine (AMD Renoir) so far.